cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,13 @@
 
 set -ex
 
+# Configure IPv6 DHCP
```

Comment:
What happens if you do not include this change at all?
I am asking to understand the results of the default behavior.

Reply:
without it the eth0 wont get this ip
```
    inet6 fd00::101/128 scope global deprecated dynamic noprefixroute 
       valid_lft 7316sec preferred_lft 0sec
```

also the block solve problems like this one
https://github.com/kubevirt/kubevirtci/pull/313#issuecomment-602570373